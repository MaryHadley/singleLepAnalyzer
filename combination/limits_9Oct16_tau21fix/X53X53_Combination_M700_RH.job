universe = vanilla
Executable = X53X53_Combination_M700_RH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M700_RH.out
Error = X53X53_Combination_M700_RH.err
Log = X53X53_Combination_M700_RH.log
Queue 1