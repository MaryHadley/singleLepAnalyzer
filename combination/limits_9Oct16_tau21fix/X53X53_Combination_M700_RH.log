000 (1295361.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295361.000.000) 10/09 22:41:05 Job executing on host: <192.168.63.226:40730>
...
006 (1295361.000.000) 10/09 22:41:14 Image size of job updated: 219188
	116  -  MemoryUsage of job (MB)
	117988  -  ResidentSetSize of job (KB)
...
006 (1295361.000.000) 10/09 22:46:15 Image size of job updated: 283332
	141  -  MemoryUsage of job (MB)
	144036  -  ResidentSetSize of job (KB)
...
006 (1295361.000.000) 10/09 23:41:22 Image size of job updated: 283332
	141  -  MemoryUsage of job (MB)
	144120  -  ResidentSetSize of job (KB)
...
006 (1295361.000.000) 10/10 14:22:56 Image size of job updated: 297956
	158  -  MemoryUsage of job (MB)
	161420  -  ResidentSetSize of job (KB)
...
005 (1295361.000.000) 10/10 14:28:31 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 15:46:06, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 15:46:06, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1592  -  Run Bytes Sent By Job
	306  -  Run Bytes Received By Job
	1592  -  Total Bytes Sent By Job
	306  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31877        1   3824811
	   Memory (MB)          :      158     3072      3072
...
