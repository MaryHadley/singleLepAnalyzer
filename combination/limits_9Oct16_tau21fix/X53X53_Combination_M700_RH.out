/var/lib/condor/execute/dir_18433
Total number of likelihood evaluations: 1100000100
Total number of likelihood evaluations: 11000100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_RH/quant-toys:0-sig-28c9b5e903.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_RH/quant-data-sig-5fd917505b.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M700_RH
