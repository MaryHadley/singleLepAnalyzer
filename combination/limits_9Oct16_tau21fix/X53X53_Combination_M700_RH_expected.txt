# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.10546   0.052586    0.22226  0.072448    0.15655
